runtime/gc_vlhgc/AllocationContextBalanced.cpp

```diff
@@ -1103,6 +1103,56 @@ MM_AllocationContextBalanced::setNumaAffinityForThread(MM_EnvironmentBase *env)
 }
 
 #if defined(J9VM_GC_SPARSE_HEAP_ALLOCATION)
+bool
+MM_AllocationContextBalanced::allocateFromSharedArrayReservedRegion(MM_EnvironmentBase *env, uintptr_t fraction)
+{
+	const uintptr_t regionSize = _heapRegionManager->getRegionSize();
+	Assert_MM_true((regionSize > fraction) && (0 != fraction));
+
+	bool shouldAllocateNewSharedRegion = false;
+
+	lockCommon();
+	uintptr_t oldBytesUsed = _sharedArrayReservedRegionsBytesUsed;
```

Comment:
you can remove oldBytesUsed (same as you don't use it in recycleToSharedArrayReservedRegion)